xmtp_content_types/src/utils.rs

```diff
@@ -0,0 +1,5 @@
+use std::collections::HashMap;
+
+pub(crate) fn get_param_or_default<'a>(params: &'a HashMap<String, String>, key: &str) -> &'a str {
+    params.get(key).map(|s| s.as_str()).unwrap_or("")
```

Comment:
I understand the value of failing open, but would it be worth logging something if this is empty so that it can be debugged after-the-fact? Just afraid of us missing mistakes here